src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -195,6 +209,14 @@ public static enum TokenAuthMethod {
      */
     private boolean nonceDisabled = false;
 
+    /** Additional number of seconds to add to token expiration
+     */
+    private Long allowedTokenExpirationClockSkewSeconds = 60L;
+
+    /** Flag to enable strict token expiration check
+     */
+    private boolean strictTokenExpirationCheck = false;
```

Comment:
On second thought, I'd pefer this option to by true by default or even renamed to disableTokenExpirationCheck.

IMHO it is better to enforce security by default and allow user to bypass it if they want (unless it is added security or it breaks usability too much - like PKCE).